src/Datadog.Trace.ClrProfiler.Managed/Helpers/AsyncHelper.cs

```diff
@@ -5,22 +5,26 @@
 
 namespace Datadog.Trace.ClrProfiler.Integrations
```

Comment:
Nit: can you fix the namespace while you're here?
```suggestion
namespace Datadog.Trace.ClrProfiler.Helpers
```